Formula/libgweather.rb

```diff
@@ -1,9 +1,9 @@
 class Libgweather < Formula
   desc "GNOME library for weather, locations and timezones"
   homepage "https://wiki.gnome.org/Projects/LibGWeather"
-  url "https://download.gnome.org/sources/libgweather/3.36/libgweather-3.36.1.tar.xz"
-  sha256 "de2709f0ee233b20116d5fa9861d406071798c4aa37830ca25f5ef2c0083e450"
-  revision 2
+  url "https://download.gnome.org/sources/libgweather/40/libgweather-40.0.tar.xz"
+  sha256 "ca4e8f2a4baaa9fc6d75d8856adb57056ef1cd6e55c775ba878ae141b6276ee6"
+  license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
```

Comment:
Repo is GPLv2: https://gitlab.gnome.org/GNOME/libgweather

Some Red Hat code is LGPLv2.1+, e.g.
https://gitlab.gnome.org/GNOME/libgweather/-/blob/40.0/libgweather/gweather-location-entry.c
```
 * This library is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public License
 * as published by the Free Software Foundation; either version 2.1 of
 * the License, or (at your option) any later version.
```

Other code is GPLv2+, e.g.
https://gitlab.gnome.org/GNOME/libgweather/-/blob/40.0/libgweather/gweather-enums.h
```
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License as
 * published by the Free Software Foundation; either version 2 of the
 * License, or (at your option) any later version.
```